integrations/schemas/repository-configuration-schema.json

```diff
@@ -158,6 +161,9 @@
                   "homepage_url": {
                     "type": "string"
                   },
+                  "purl": {
+                    "type": "string"
+                  },
```

Comment:
@MarcelBochtler how could this be missing if the file was auto-generated?

Reply:
It was mostly auto-generated.
I had to remove some values and I guess `purl` was removed by accident.